am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/importer/DefaultPermissionIntegrationTest.java

```diff
@@ -45,31 +49,31 @@ void setUp() {
     @Test
     void shouldNotBeAbleToCreateDefaultPermissionWhenRoleDoesNotExist() {
         assertThatExceptionOfType(PersistenceException.class)
-            .isThrownBy(() ->
-                service.grantDefaultPermission(createDefaultPermissionGrant(resourceDefinition, ImmutableSet.of(READ))))
+            .isThrownBy(() -> service.grantDefaultPermission(createDefaultPermissionGrant(
+                resourceDefinition, ImmutableSet.of(READ), "", "Solicitor")))
             .withMessageContaining("(role_name)=(Solicitor) is not present in table \"roles\"");
```

Comment:
it would be cool if we could inject the UUID into the error message here. Unsure how off the top of my head

Reply:
👍